ethcore/src/snapshot/mod.rs

```diff
@@ -119,6 +119,7 @@ pub struct Progress {
 	blocks: AtomicUsize,
 	size: AtomicUsize, // Todo [rob] use Atomicu64 when it stabilizes.
```

Comment:
not introduced by the PR, but `AtomicU64` is [stable](https://doc.rust-lang.org/std/sync/atomic/struct.AtomicU64.html) now

Reply:
Is it obvious to you why `AtomicU64` is better? Just avoid ending up with an unexpected u32?